### PR TITLE
add new APIs to construct model_data by hand

### PR DESCRIPTION
**Pull request by okdshin**
- menoh_make_model_data()
- menoh_model_data_add_new_node()
- menoh_model_data_add_input_name_to_current_node()
- menoh_model_data_add_output_name_to_current_node()
- menoh_model_data_add_attribute_int_to_current_node()
- menoh_model_data_add_attribute_float_to_current_node()
- menoh_model_data_add_attribute_ints_to_current_node()
- menoh_model_data_add_attribute_floats_to_current_node()
- ~~menoh_model_data_add_initializer()~~ menoh_model_data_add_parameter()

### CODE REVIEW

File: menoh/menoh.cpp [hunk 1]
```diff
@@ -102,6 +102,92 @@ menoh_error_code menoh_make_model_data_from_onnx_data_on_memory(
         return menoh_error_code_success;
     });
 }
+menoh_error_code menoh_make_model_data(menoh_model_data_handle* dst_handle) {
+    return check_error([&]() {
+        *dst_handle = std::make_unique<menoh_model_data>().release();
```
Comment (msakai) [inline menoh/menoh.cpp]: Is it common in C++ to just construct a value using `std::make_unique` and immediately call `release`?

Comment (okdshin) [reply menoh/menoh.cpp]: It is paranoiac code to avoid `new` keyword. I have no idea that it is common but we can avoid `new`.

Comment (msakai) [reply menoh/menoh.cpp]: I see.

**Comment by okdshin** [comment]
I see. I will modify.

**Comment by msakai** [comment]
Hmm, I reconsidered about it and maybe it will be enough to add a API like `menoh_model_data_add_new_graph`  for adding a new graph and switching context to the new graph (just like `menoh_model_data_add_new_node`).
If that is the case, we don't need to change the API for now.

**Comment by okdshin** [comment]
I see. So I will merge anyway.

**Pull request merged**